server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/routes/JMAPApiRoutes.scala

```diff
@@ -29,21 +29,26 @@ object JMAPApiRoutes {
 }
 
 class JMAPApiRoutes {
```

Comment:
In theory it should extend `JMAPRoutes` from the Y. I know it's still not finalised, but you can still reuse the current state of #3211 as a no review commit? It should be enough for now to get started on this

Reply:
yes, i will rebase it later.I just do unit test to my not integration test yet.